TEMPLATE.md

```diff
@@ -0,0 +1,652 @@
+## Shortcuts from raulfraile, applied on Phpstorm by Pouzor ##
+
+All shortcuts start with the `sf` prefix and are both short and intuitive:
+
+### Controller ###
+
+`sfcontroller`
+
+``` php
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    public function indexAction()
+    {
+        return $this->render('.html.twig');
+    }
+}
+```
+
+`sfcontrollera`
+
+``` php
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="")
+     * @Template()
```

Comment:
We have to be opinionated, and my opinion is that `@Route` is great, and `@Template` should not be included